ChainReactionBindings/Bindings/WishlistPopupPageBindings.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using ChainReactionBindings.TestBase.Pages;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Support.UI;
+using TechTalk.SpecFlow;
+
+namespace ChainReactionBindings.Bindings
+{
+	[Binding]
+	class WishlistPopupPageBindings
+	{
+		WishlistPopupPage page = new WishlistPopupPage();
+
+		[Then(@"I see add to wishlist popup is loaded")]
+		public void ThenISeeAddToWishlistPopupIsLoaded()
+		{
+			page.WishlistPopupPageLoaded();
+			//Assert.IsTrue(page.AddPopupElement.Displayed);
+		}
+
+		[When(@"I choose first good and click Add To Wishlist button")]
+		public void ChooseFirstGoodAndClickAddToWishlistButton()
+		{
+			page.AddToWishlistInPop();
+		}
+
+		[Then(@"I see select wishlist popup is loaded")]
+		public void ThenISeeSelectWishlistPopupIsLoaded()
+		{
+			page.SelectPopElement.WaitElementAppears(10);
+			Assert.IsTrue(page.SelectPopElement.Displayed);

```

Comment:
We can delete this line, WaitElementAppears already check for Displayed and will throw exception if it's false